test/unit/getAuthenticatorFromEnvCloudantExtension.test.js

```diff
@@ -21,30 +21,41 @@ const { CouchdbSessionAuthenticator } = require('../../index.ts');
 
 describe('Test getAuthenticatorFromEnvCloudantExtension', () => {
   it('Create couchdb_session authenticator', () => {
-    process.env.TEST_AUTH_TYPE = 'couchdb_session';
-    process.env.TEST_USERNAME = 'username';
-    process.env.TEST_PASSWORD = 'password';
-    const auth = getAuthenticatorFromEnvCloudantExtension('test');
+    process.env.TEST1_AUTH_TYPE = 'couchdb_session';
```

Comment:
Just a nit: with an `afterEach` func where all env vars starting with `TEST_` are cleared could have prevented this whole refactor, plus if we would like to extend tests in the future, we will not remember to use other `TEST{N}` variables and make a mistake again.

Reply:
So previously there were 3 environment variables and now there are 12. I am +1 to this refactor suggestion.

Reply:
This is a valid point and to be fair I've been on the fence for some time trying to decide how I want to refactor this, with `afterEach` or with individual var renames. I went with the latter for two reasons:

1. We are probably not going to touch this test again, so I think it's more important to keep it more readable than more extendable. I, as a rule of thumb, prefer to keep my tests "dumb" rather than DRY - tests are boring, I'd rather be able to grasp and change them quickly instead of trying to remember how this particular utest framework works or what subtle thing is going in the code. And it's not that we are short on env vars, so we can't spare a dozen 😉 
2. This is kind of how core does this and I suspect for the same reasons. Surprisingly they are not consistent, so this is most representative in java, not in node https://github.com/IBM/java-sdk-core/blob/main/src/test/java/com/ibm/cloud/sdk/core/util/CredentialUtilsTest.java#L46